### ISSUE TITLE
KBVE x WebSite x (Tonic-AI) Stack

### ISSUE THREAD

**Issue by Josephrp** [Tonic-AI/tonic-ai-website#1]
@h0lybyte + @twilwa , please use the following [doodle](https://doodle.com/meeting/participate/id/aAQJzk7a) to set a time and get the next step done :-)

**Comment by h0lybyte** [comment]
Yeah @twilwa what time are you streaming this week or weekend? We could do another live stream style session around the setup of the mono repo and tonic-ai.

Before we get started, what sub domain do we want to use? We can do a couple test runs as well, kinda like how we did Supportdate!
